### ISSUE TITLE
PowerShell-Core tries to load Registry on Linux

### ISSUE THREAD

**Issue by knoxyca** [Jaykul/Jupyter-PowerShell#12]
When launching PowerShell-Core notebook on Linux throws an exception about not supporting Registry on linux:

```
PS /home/corbob/jupyter> jupyter notebook                                                                                                                                
[I 22:02:51.651 NotebookApp] Serving notebooks from local directory: /home/corbob/jupyter
[I 22:02:51.651 NotebookApp] 0 active kernels 
[I 22:02:51.651 NotebookApp] The Jupyter Notebook is running at: http://localhost:8888/?token=cb0aee7035533fe2a5982ed0d2ced625b2eefc825fc107e3
[I 22:02:51.651 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 22:02:51.652 NotebookApp] 
    
    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://localhost:8888/?token=cb0aee7035533fe2a5982ed0d2ced625b2eefc825fc107e3
Created new window in existing browser session.
[9707:9733:0703/220252.035095:ERROR:browser_gpu_channel_host_factory.cc(103)] Failed to launch GPU process.
[I 22:02:52.048 NotebookApp] Accepting one-time-token-authenticated connection from ::1
[I 22:02:57.337 NotebookApp] Creating new notebook in 
[W 22:02:58.076 NotebookApp] 404 GET /nbextensions/widgets/notebook/js/extension.js?v=20170703220251 (::1) 10.93ms referer=http://localhost:8888/notebooks/Untitled.ipynb?kernel_name=powershell-core
[I 22:02:58.149 NotebookApp] Kernel started: e696de97-1220-46b1-86cf-16d220113188

Unhandled Exception: System.TypeInitializationException: The type initializer for 'System.Management.Automation.PSVersionInfo' threw an exception. ---> System.TypeInitializationException: The type initializer for 'Microsoft.Win32.Registry' threw an exception. ---> System.PlatformNotSupportedException: Registry is not supported on this platform.
   at Microsoft.Win32.RegistryKey.OpenBaseKeyCore(RegistryHive hKey, RegistryView view)
   at Microsoft.Win32.Registry..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.PSVersionInfo.GetWSManStackVersion()
   at System.Management.Automation.PSVersionInfo..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.PSVersionInfo.get_PSVersion()
   at System.Management.Automation.PSSnapInReader.ReadRegistryInfo(Version& assemblyVersion, String& publicKeyToken, String& culture, String& architecture, String& applicationBase, Version& psVersion)
   at System.Management.Automation.PSSnapInReader.ReadEnginePSSnapIns()
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault()
   at Jupyter.PowerShell.PowerShellEngine..ctor(ILogger logger)
   at Jupyter.PowerShell.Kernel.Main(String[] args)
```

**Comment by knoxyca** [comment]
It looks like this might be related to what is published to the myget repository?

Copying the entirety of /opt/microsoft/powershell/<version> to the PowerShell-Core kernel directory results in the kernel working. For whatever reason copying the different files, it doesn't throw an exception, but times out when trying to communicate and doesn't actually execute any code.

Files copied from PowerShell install directory:
```
Microsoft.CodeAnalysis.CSharp.dll
Microsoft.CodeAnalysis.dll
Microsoft.PowerShell.Commands.Management.dll
Microsoft.PowerShell.Commands.Utility.dll
Microsoft.PowerShell.ConsoleHost.dll
Microsoft.PowerShell.CoreCLR.AssemblyLoadContext.dll
Microsoft.PowerShell.CoreCLR.Eventing.dll
Microsoft.PowerShell.Security.dll
Microsoft.Win32.Registry.AccessControl.dll
Newtonsoft.Json.dll
System.Data.SqlClient.dll
System.IO.Packaging.dll
System.Net.Http.WinHttpHandler.dll
System.Security.Cryptography.Pkcs.dll
System.ServiceModel.Duplex.dll
System.ServiceModel.Http.dll
System.ServiceModel.NetTcp.dll
System.ServiceModel.Primitives.dll
System.ServiceModel.Security.dll
System.ServiceProcess.ServiceController.dll
System.Text.Encoding.CodePages.dll
System.Text.Encodings.Web.dll
System.Threading.AccessControl.dll
libpsl-native.so #copied based on exception after System.Management.Automation.dll copied
System.Management.Automation.dll #first one copied based on initial exception
```

**Comment by vors** [comment]
Hitting the same now, because of this line
https://github.com/PowerShell/PowerShell/blob/beffdcf94d00b51d9339bb26dbc0e477cd147c92/src/System.Management.Automation/CoreCLR/CorePsPlatform.cs#L78

The root cause of it is https://github.com/PowerShell/PowerShell/issues/5966 which is actually a dupe of a long-standing issue of publishing windows only packages.

**Comment by Jaykul** [comment]
This is fixed in the 1.0.0 release. Thanks for the help @vors
